source/systemUtils.py

```diff
@@ -166,9 +166,7 @@ def _getDesktopName() -> str:
 
 
 def _isSecureDesktop() -> bool:
-	"""
-	When NVDA is running on a secure screen,
-	it is running on the secure desktop.
+	"""	When NVDA is running on a secure screen, it is running on the secure desktop.
```

Comment:
Fix lint
```suggestion
	"""
	When NVDA is running on a secure screen, it is running on the secure desktop.
```

Reply:
Can I ask what the lint error is in these?

My understanding was that we preferred the opening line of docstring on same line as docstring quotes method, which is listed as optional for [Multi-line Docstrings in PEP-257](https://peps.python.org/pep-0257/#multi-line-docstrings). We have nothing in tests/lint/readme.md either way, though I thought we used to have it written up somewhere that this was the NVDA choice. That was why I changed those. I'm happy to change them back, but I would like to understand why I'm doing it.

Examples of that style in core: `gui.AddonsDialog._instance`, `inputCore.InputGesture`, `inputCore.InputGesture.reportExtra`. Just a few of many.

Reply:
The lint issue is a tab prefacing the start of the string.
I think we are agnostic as to whether you start the text on the same line or not, but I think we more commonly do it on a new line.